js-intermediate.test.js

```diff
@@ -3,24 +3,50 @@
 
 // 1. Write the code that would make this test pass.
 
-describe("multTwo", () => {
-    test("returns an array with all the numbers multiplied by two", () => {
-        expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
-        expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
-        expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
-    })
-})
+// describe("multTwo", () => {
+//     test("returns an array with all the numbers multiplied by two", () => {
+//         expect(multTwo([3, 4, 5])).toEqual([6, 8, 10])
+//         expect(multTwo([23, -9, 0])).toEqual([46, -18, 0])
+//         expect(multTwo([4.5, -4.5, 12])).toEqual([9, -9, 24])
+//     })
+// })
+
+var arr1 = [3, 4, 5]
+var arr2 = [23, -9, 0]
+var arr3 = [4.5, -4.5, 12]
+
+const multTwo = (arr) => {
+  let newArr = arr.map(value => value * 2)
+  return newArr
+}
+
+console.log(multTwo(arr1))
+console.log(multTwo(arr2))
+console.log(multTwo(arr3))
 
```

Comment:
When coding in a test file, you don't need to create variables or log your function. The test takes care of that for you.